app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,14 @@
+class MoviesController < ApplicationController
+
+def index 
+  @movies = Movie.all
+  @screenings = Screening.all
+end
+
+def show
+@movie = Movie.find(params[:id])
+@screenings = @movie.screenings
+
+end
+
```

Comment:
Please try to format your code 🙏  here are some examples https://github.com/rubocop/ruby-style-guide 

```suggestion
class MoviesController < ApplicationController
  def index
    @movies = Movie.all
    @screenings = Screening.all
  end

  def show
    @movie = Movie.find(params[:id])
    @screenings = @movie.screenings
  end
end
```